src/curve/stable/StablePair.sol

```diff
@@ -356,6 +338,7 @@ contract StablePair is ReservoirPair {
         return _getCurrentAPrecise();
     }
 
+    // TODO: Is this function needed?
     function getAmountOut(address tokenIn, uint256 amountIn) public view returns (uint256 finalAmountOut) {
```

Comment:
technically no since the periphery's quoter has functions to facilitate this. In this core repo we have a few tests that use it but can be replaced